core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -305,39 +305,36 @@ service MgmtPublicService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Utils"};
   }
 
-  // List pipeline triggers
+  // Get pipeline trigger count
   //
-  // Returns a paginated list of pipeline executions.
-  rpc ListPipelineTriggerRecords(ListPipelineTriggerRecordsRequest) returns (ListPipelineTriggerRecordsResponse) {
-    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/triggers"};
+  // Returns the pipeline trigger count of a given requester within a timespan.
+  // Results are grouped by trigger status.
+  rpc GetPipelineTriggerCount(GetPipelineTriggerCountRequest) returns (GetPipelineTriggerCountResponse) {
+    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/trigger-count"};
+    option (google.api.method_signature) = "requester,start,stop";
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
   }
 
-  // List pipeline trigger metrics
+  // List pipeline trigger time charts
   //
-  // Returns a paginated list of pipeline executions aggregated by pipeline ID.
-  rpc ListPipelineTriggerTableRecords(ListPipelineTriggerTableRecordsRequest) returns (ListPipelineTriggerTableRecordsResponse) {
-    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/tables"};
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
-  }
-
-  // List pipeline trigger computation time charts
-  //
-  // Returns a paginated list with pipeline trigger execution times, aggregated
-  // by pipeline and time frames.
+  // Returns a timeline of pipline trigger counts for a given requester. The
+  // response will contain one set of records (datapoints), representing the
+  // amount of triggers in a time bucket.
   rpc ListPipelineTriggerChartRecords(ListPipelineTriggerChartRecordsRequest) returns (ListPipelineTriggerChartRecordsResponse) {
     option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/charts"};
+    option (google.api.method_signature) = "requester,aggregation_window,start,stop";
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
   }
 
   // List Instill Credit consumption time charts
   //
   // Returns a timeline of Instill Credit consumption for a given owner. The
-  // timeline consists of a list of time frames that contain the aggregated
-  // credit consumption.
+  // response will contain one set of records (datapoints) per consumption
+  // source (e.g. "pipeline", "model"). Each datapoint represents the amount
+  // consumed in a time bucket.
   rpc ListCreditConsumptionChartRecords(ListCreditConsumptionChartRecordsRequest) returns (ListCreditConsumptionChartRecordsResponse) {
     option (google.api.http) = {get: "/v1beta/metrics/credit/charts"};
-    option (google.api.method_signature) = "owner,aggregation_window,filter";
+    option (google.api.method_signature) = "owner,aggregation_window,start,stop";
```

Comment:
BTW, @jvallesm
Do you know how the `method_signature` works exactly? When I worked on another task, I read the documentation and did some experiments, but the changed value didn't seem to affect any generated code.

Reply:
I can only find the [Java client](https://cloud.google.com/java/docs/reference/proto-google-common-protos/2.9.5/com.google.api.ClientProto#com_google_api_ClientProto_methodSignature) implementing the `method_signature`. I'm thinking we can remove this option from  our protobufs to reduce maintenance effort.

Reply:
➕ I couldn't see any impact on the generated code either, I was just maintaining it because it was in place. I'll remove these options from the api 👍